be/src/exprs/agg/window.h

```diff
@@ -128,7 +128,11 @@ class ValueWindowFunction : public WindowFunction<State> {
 
         InputColumnType* column = down_cast<InputColumnType*>(data_column);
         for (size_t i = start; i < end; ++i) {
-            column->get_data()[i] = value;
+            if constexpr (PT != TYPE_HLL && PT != TYPE_OBJECT) {
```

Comment:
> What if some other data types ? How to make it compatible with all types?

Because for ObjectColumn(HLL and BITMAP), we can't write datas through get_data(it juse return _cache). 
We should use get_object to write datas directly.